crates/storage/db-common/src/init.rs

```diff
@@ -138,6 +143,9 @@ where
 
     insert_genesis_state(&provider_rw, alloc.iter())?;
 
+    // compute state root to populate trie tables
+    compute_state_root(&provider_rw)?;
```

Comment:
-.-

this only affected big ass genesis tries,
I guess this would have also been an issue for bloatnet eventually?

Reply:
Only if it used --chain with a large genesis json, init from state dump for example already calls this